src/screens/TimeZones/index.js

```diff
@@ -1,16 +1,58 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import i18next from 'i18next';
+import { create, color } from '@amcharts/amcharts4/core';
+import {
+  MapChart,
+  projections,
+  MapPolygonSeries,
+} from '@amcharts/amcharts4/maps';
+import worldTimeZoneAreasHigh from '@amcharts/amcharts4-geodata/worldTimeZoneAreasHigh';
+
+import { THEME_COLORS } from '@constants/colors';
+import { MOCK_DATA } from './constants';
+import styles from './styles.module.scss';
 
 function TimeZones() {
+  useEffect(() => {
+    let map = create('mapRef', MapChart);
```

Comment:
Since this code doesn't depend on props or state I'd extract it to an external function either in utils or in this file but outside the component and just call it in the useEffect